node-unix-interface.md

```diff
@@ -0,0 +1,11 @@
+# The Node-Unix interface
+
+Imagine being able to build scripts that can fit into any pipeline of Unix
+commands at any place. Not only is that possible with Node.js, it's easy! Node
+provides easy access to everything you need to create powerful system level
+utilities, and following the Unix philosophy of modularity and reusability
```

Comment:
Consider swapping this and what aspects of Node you'll cover specifically to allow CLI app development with Node as it ties into what you say 2 lines up.